### PR TITLE
Move MAV_CMD_DO_AUTOTUNE_ENABLE to common

### PR DESCRIPTION
**Pull request by jkflying**
We'll be bringing autotune capabilities to PX4 as well, so to avoid fragmentation I'd like to bring this message to common.

### CODE REVIEW

**Comment by hamishwillee** [comment]
Excellent. 

@auturgy Any objection before I merge (it is unchanged from ArduPilot version).
FMI is Autotuning a mode? i.e. how does GCS know when the task is finished?

**Comment by jkflying** [comment]
Still working out the exact message flow we want, but it runs in a few seconds so the UX considerations are pretty different from previous implementations.

File: message_definitions/v1.0/ardupilotmega.xml [hunk 1]
```diff
@@ -27,17 +27,7 @@
...
-        <description>Enable/disable autotune.</description>
-        <param index="1" label="Enable" minValue="0" maxValue="1" increment="1">Enable (1: enable, 0:disable).</param>
-        <param index="2">Empty.</param>
-        <param index="3">Empty.</param>
-        <param index="4">Empty.</param>
-        <param index="5">Empty.</param>
-        <param index="6">Empty.</param>
-        <param index="7">Empty.</param>
-      </entry>
+      <!-- 214 used by common.xml -->
```
Comment (auturgy) [inline message_definitions/v1.0/ardupilotmega.xml:30]: 212?

**Comment by auturgy** [comment]
Looks like a little typo, but beyond that I'm all for standardising

File: message_definitions/v1.0/ardupilotmega.xml (hunk 1 above)
Comment (jkflying) [reply message_definitions/v1.0/ardupilotmega.xml:30]: Maybe I missed the convention here, the intent is more like "0 - 214 used by common.xml". Want me to change it?

Comment (hamishwillee) [reply message_definitions/v1.0/ardupilotmega.xml]: @jkflying the convention is/has been that when an id moves out of a dialect we list it here. You moved 212 out but added note for 214.
But in this case your suggestion is much better, and I have changed it.

**Comment by hamishwillee** [comment]
As discussed in the dev call, I will merge. 

Given there is work going on here it would be good to know how ArduPilot knows when the tuning is finished so I can document this and avoid spec ambiguity. It could be that you monitor for a mode change, or maybe this is a blocking command, so you use long running command with progress notifications.  @meee1 @auturgy If you could confirm would be great.

**Pull request merged**